packages/cli/src/ActiveWebhooks.ts

```diff
@@ -0,0 +1,161 @@
+import { Service } from 'typedi';
+import type { Response } from 'express';
+import { Workflow, NodeHelpers } from 'n8n-workflow';
+import type { INode, IWebhookData, IHttpRequestMethods } from 'n8n-workflow';
+
+import { WorkflowRepository } from '@db/repositories/workflow.repository';
+import type {
+	IResponseCallbackData,
+	IWebhookManager,
+	WebhookAccessControlOptions,
+	WebhookRequest,
+} from '@/Interfaces';
+import { Logger } from '@/Logger';
+import { NodeTypes } from '@/NodeTypes';
+import { WebhookService } from '@/services/webhook.service';
+import { WebhookNotFoundError } from '@/errors/response-errors/webhook-not-found.error';
+import { NotFoundError } from '@/errors/response-errors/not-found.error';
+import * as WorkflowExecuteAdditionalData from '@/WorkflowExecuteAdditionalData';
+import * as WebhookHelpers from '@/WebhookHelpers';
+import { WorkflowStaticDataService } from '@/workflows/workflowStaticData.service';
+
+@Service()
+export class ActiveWebhooks implements IWebhookManager {
```

Comment:
We have four kinds of webhooks (test, waiting, production, form) but only production and form webhooks rely on `ActiveWebhooks`. I find this a bit misleading, because all four webhook kinds can be active.

Can we think of a better name to group production and form webhooks? `RegularWebhooks`?

Reply:
Forms have their own test, wait, & production urls, and I'm planning to move forms into their own separate classes, instead of overloading webhooks like we currently do. so, we only need to care about webhooks when naming this.